src/element.c

```diff
@@ -246,16 +246,13 @@ stumpless_element_to_string( const struct stumpless_element *element ) {
 
     unlock_element( element );
 
-    format[0] = '<';
-    format[name_len + 1] = '>';
-
     if (param_count != 0 ) {
-      // <name>:[param_1_to_string,param_2_to_string,etc.] (with params)
-      format[name_len + 2] = ':';
-      format[name_len + 3] = '[';
+      // name=[param_1_to_string,param_2_to_string,etc.] (with params)
+      format[name_len ] = '=';
+      format[name_len + 1] = '[';
       format[pos_offset] = ']';
     } else {
-      // <name> (no params)
+      // name (no params)
       // pos_offset is name_len + 4 here
       pos_offset -= 3;
```

Comment:
This offset needs to be adjusted to account for the change in character position.

Reply:
Is it correct now?

Reply:
It seems like there are still some tests failing, which may be because some of these counts are still off.

Reply:
Is there any way to verify the changes locally? The build test runs fine, so I am unable to tweak it.

Reply:
I recommend you look through the failed CI checks, particularly for the MacOS builds, to see what tests failed there. Different compilers have different sanitization strategies, so what works in GCC may not work in clang, for example. You can also try valgrind, which is also failing in the Linux CI runs, which will point you at more specific memory access violations.

Reply:
If you want to run the checks locally (which I definitely recommend), you can use `make check` to build all of the tests first. Then, use `valgrind` to run the specific test that is failing in the CI. You can see how the CI tests run this in the `.github/workflows/linux.yml` file around [line 56](https://github.com/goatshriek/stumpless/blob/latest/.github/workflows/linux.yml#L56) as an example. Once you make changes to the test/source, don't forget to re-run the make command to rebuild the tests with your changes.

Reply:
I get this when I try to run Valgrind, any ideas on whats causing this?

```
==11209== Memcheck, a memory error detector
==11209== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==11209== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==11209== Command: ./function-test-buffer
==11209== 
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x23
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x25
### unhandled dwarf2 abbrev form code 0x23
==11209== Valgrind: debuginfo reader: ensure_valid failed:
==11209== Valgrind:   during call to ML_(img_get)
==11209== Valgrind:   request for range [1956341782, +4) exceeds
==11209== Valgrind:   valid image size of 2202248 for image:
==11209== Valgrind:   "/home/rmknan/Github/Stump/build/function-test-buffer"
==11209== 
==11209== Valgrind: debuginfo reader: Possibly corrupted debuginfo file.
==11209== Valgrind: I can't recover.  Giving up.  Sorry.
==11209== 
```

Reply:
I am not sure what would cause that offhand. Perhaps a difference in valgrind debug info support and the compiler used?

Reply:
How would I check if these two are the same or not? and please be as detailed as possible. Thank you.

Reply:
I'm not sure how you would check compatibility between valgrind and the dwarf info. A search for part of the error messages turns up [this thread](https://github.com/llvm/llvm-project/issues/56550), which might be worth trying. You can set the flags during compilation by setting the `CFLAGS` environment variable before running cmake, if you need to.